app_feup/lib/controller/notifications/local_notifications.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:io';
```

Comment:
I see a lot of android specific stuff in this module. Is this supposed to not work in iOS?